CMakeLists.txt

```diff
@@ -377,54 +389,14 @@ set(MKDIR_OPTS "")
 if(WIN32)
   set(PROCESS_FAMILY "windows")
 
-  set(WINDOWS_DEP_DIR "$ENV{ChocolateyInstall}/lib")
-  string(REGEX REPLACE "\\\\" "/" WINDOWS_DEP_DIR ${WINDOWS_DEP_DIR})
-  set(BOOST_DEP_DIR "$ENV{HOMEDRIVE}/local/boost-msvc14")
-
-  # Win64 specific BZIP2 variables
-  set(BZIP2_INCLUDE_DIR "${WINDOWS_DEP_DIR}/bzip2/local/include")
-  set(BZIP2_LIBRARIES "${WINDOWS_DEP_DIR}/bzip2/local/lib/libbz2.lib")
-
-  # Win64 specific OpenSSL variables
-  set(OPENSSL_ROOT_DIR "${WINDOWS_DEP_DIR}/openssl/local/lib")
-  set(OPENSSL_INCLUDE_DIR "${WINDOWS_DEP_DIR}/openssl/local/include")
-  set(OPENSSL_CRYPTO_LIBRARY "${WINDOWS_DEP_DIR}/openssl/local/lib/libeay32.lib")
-  set(OPENSSL_SSL_LIBRARY "${WINDOWS_DEP_DIR}/openssl/local/lib/ssleay32.lib")
-  set(OPENSSL_LIBRARIES ${OPENSSL_SSL_LIBRARY} ${OPENSSL_CRYPTO_LIBRARY})
-  set(OPENSSL_USE_STATIC_LIBS TRUE)
-
-  # Win64 specific Thrift
-  set(THRIFT_INCLUDE_DIR "${WINDOWS_DEP_DIR}/thrift-dev/local/include")
-  set(thrift_library "${WINDOWS_DEP_DIR}/thrift-dev/local/lib/thriftmd.lib")
-  set(THRIFT_COMPILER "${WINDOWS_DEP_DIR}/thrift/tools/thrift.exe")
-
-  # Win64 specific rocksdb
-  set(ROCKSDB_STATIC_LIBRARY "${WINDOWS_DEP_DIR}/rocksdb/local/lib/rocksdblib.lib")
-  set(ROCKSDB_SNAPPY_LIBRARY "${WINDOWS_DEP_DIR}/snappy-msvc/tools/snappy-windows-1.1.1.8/native/snappy64.lib")
-
-  # Win64 specific doxygen
-  set(DOXYGEN_EXECUTABLE "${WINDOWS_DEP_DIR}/doxygen/tools/doxygen.exe")
-
-  # Win64 specific boost
-  set(boost_filesystem_library "${BOOST_DEP_DIR}/lib64-msvc-14.0/libboost_filesystem-vc140-mt-1_59.lib")
-  set(boost_system_library "${BOOST_DEP_DIR}/lib64-msvc-14.0/libboost_system-vc140-mt-1_59.lib")
-  set(boost_regex_library "${BOOST_DEP_DIR}/lib64-msvc-14.0/libboost_regex-vc140-mt-1_59.lib")
-
   # Win64 specific gflags
   set(BUILD_SHARED_LIBS ON CACHE BOOL "Build shared gtest library" FORCE)
-  set(gflags_library "${WINDOWS_DEP_DIR}/gflags-dev/local/lib/gflags_static.lib")
-
-  # Win64 specific glog
-  set(glog_library "${WINDOWS_DEP_DIR}/glog/local/lib/glog.lib")
 
-  # Win64 specific cpp-netlib library
-  set(cppnetlib-uri_library "${WINDOWS_DEP_DIR}/cpp-netlib/local/lib/cppnetlib-uri.lib")
-  set(cppnetlib-client-connections_library "${WINDOWS_DEP_DIR}/cpp-netlib/local/lib/cppnetlib-client-connections.lib")
-
-
-  include_directories("${THRIFT_INCLUDE_DIR}")
-  include_directories("${BOOST_DEP_DIR}")
-  include_directories("${BOOST_DEP_DIR}/boost")
+  include_directories("$ENV{HOMEDRIVE}/local/boost-msvc14/boost")
+  include_directories("$ENV{HOMEDRIVE}/local/boost-msvc14")
+  include_directories("${WINDOWS_DEP_DIR}/bzip2/local/include")
```

Comment:
There's quite a few programs that get dropped in that directory and globbing might try bringing in a few directories that don't exist. How does CMake handle attempted globs on folders being non-existant? Also pulling in everything from that directory might catch other applications installed locally on the machine. What does the CMake behavior look like if we start pulling in other static libs inadvertently?